app/ember/package.json

```diff
@@ -43,8 +43,9 @@
   "peerDependencies": {
     "@babel/core": "*",
     "babel-plugin-ember-modules-api-polyfill": "^2.12.0",
-    "babel-plugin-htmlbars-inline-precompile": "2 || 3",
-    "ember-source": "^3.16.0"
+    "babel-plugin-htmlbars-inline-precompile": "2 || 3 || 4",
+    "ember-cli-htmlbars": "5",
```

Comment:
Only really needed for the `template` type returned from a story function

Reply:
This is important, although ember projects require you to have it in dependencies (or else the project is unusable). But it must be at least `>=5.2.1` (iirc 5.2 had some important change)